<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ULine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 3px solid green;
border-radius: 7px;
}
QWidget:hover{
border-color: darkgreen;
}</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 3px solid green;
border-radius: 7px;
}
QWidget:hover{
border-color: darkgreen;
}</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 3px solid green;
border-radius: 7px;
}
QWidget:hover{
border-color: darkgreen;
}</string>
    </property>
    <property name="placeholderText">
     <string>Fristname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 3px solid green;
border-radius: 7px;
}
QWidget:hover{
border-color: darkgreen;
}</string>
    </property>
    <property name="placeholderText">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border: 3px solid green;
border-radius: 7px;
}
QWidget:hover{
border-color: darkgreen;
}</string>
    </property>
    <property name="placeholderText">
     <string>Gmail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
color: white;
background-color: green;
border-radius: 7px;

}
QWidget:Hover{
background-color: darkgreen;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: white;
border: 4px solid green;
border-radius: 10px
}
QWidget:hover{
border-color: dakgreen;
}
QWidget:checked{
background-color: green;
border-color: darkgreen;
}
QWidget:hover:checked{
background-color: darkgreen;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="WomanBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: white;
border: 4px solid green;
border-radius: 10px
}
QWidget:hover{
border-color: dakgreen;
}
QWidget:checked{
background-color: green;
border-color: darkgreen;
}
QWidget:hover:checked{
background-color: darkgreen;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ManLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Man</string>
    </property>
   </widget>
   <widget class="QLabel" name="WomanLbl">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Woman</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkLabel_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Please enter your infomation</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
